resources/src/sessionCookie.php

```diff
@@ -65,7 +81,16 @@ public function headAccess()
                 include 'userdash.php';
             }
         } else {
-            include 'index.php';
+            include 'homepage.php';
```

Comment:
Please refactor so that include statements are not present in the class methods.

Reply:
@guptahemant  BUT can we keep that for later!
because this code is just working and tested.
now refactoring em will take more time which could delay others implementation